pidcat.py

```diff
@@ -334,7 +334,7 @@ def tag_in_tags_regex(tag, tags):
   linebuf = ''
 
   # right-align tag title and allocate color if needed
-  if tag != last_tag or args.always_tags:
+  if (tag != last_tag or args.always_tags) and args.tag_width > 0:
```

Comment:
The `tag_width > 0` check should wrap this whole block including the divider space that's added below. Otherwise you get a needless leading space.